src/menu/spawn_button.rs

```diff
@@ -0,0 +1,38 @@
+use bevy::prelude::*;
+
+pub fn spawn_button<T>(parent: &mut ChildBuilder, entity: T, menu_font: Handle<Font>, value: &str)
```

Comment:
takie różne "utils" to raczej do mod.rs powinny iść, a na pewno nie do osobnego pliku, bo potem trzeba go importować przez `use cośtam::spawn_button::spawn_button`